### ISSUE TITLE
[MU4 Issue] Publish Tab doesn't lock the score

### ISSUE THREAD

**Issue by oMrSmith** [musescore/MuseScore#13397]
Even if the publish tab would never get any further features,
it is a great way to get an instant PDF-preview.

However: If the publish tab is not meant for writing, which it isn't,
I think, the score should be locked - at least by default, and behave as an PDF
in order to avoid accidental changes.

**Comment by Tantacrul** [comment]
Yeah, I agree. It's a bit weird at the moment. I don't think you should be able to select anything in this view.

Incidentally, the Publish view will very much get more features! Quite soon too 😄

**Comment by oMrSmith** [comment]
> I don't think you should be able to select anything in this view.

I agree 100%.

**Comment by Jojo-Schmitz** [comment]
> Incidentally, the Publish view will very much get more features! Quite soon too 😄

3 years later aint soon.

Issue (sort of) came up again in https://musescore.org/en/node/389453

**Comment by Jojo-Schmitz** [comment]
Came up again in https://musescore.org/en/node/390011
